cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/template/etl/realtime/source/KafkaSource.java

```diff
@@ -185,5 +239,44 @@ public String getKafkaBrokers() {
     public Long getDefaultOffset() {
       return defaultOffset;
     }
+
+    private void validate() {
```

Comment:
Is there any other function need to be done to validate? If not why don't we move the check for schema null or empty inside parseSchema and call it from configurePipeline?